.tekton/main-push.yaml

```diff
@@ -245,6 +245,38 @@ spec:
       - name: basic-auth
         workspace: git-auth
 
+    # TODO(ROX-23257): remove once Central does not depend on rocksdb
+    - name: clone-rocksdb
+      params:
+      - name: url
+        value: 'https://github.com/facebook/rocksdb.git'
```

Comment:
Yes, you're right, there's a risk the repo can go away.

In downstream, [we specify the same repo](https://gitlab.cee.redhat.com/stackrox/rhacs-midstream/-/blob/08e640ab3e79131ca3eed8d759a16ccae7f9e847/distgit/containers/rhacs-main/container.yaml.in#L25-29), but there Cachito should take a never-expiring copy so that we could always rebuild, theoretically.

`git-clone` task does not seem to currently have such perpetual caching and so there's a risk our Konflux builds will stop working if `rocksdb` repo is taken down. In such case we'll have to manually create a fork under `stackrox` org from a state in Cachito.

I want to take this risk in spite of coming `rocksdb` deprecation which will remove this step. https://issues.redhat.com/browse/ROX-21207
Besides, creating the fork now may seem a bit late.

Reply:
Yeah, the risk sounds reasonable, even before you gave the explanation :).